index.js

```diff
@@ -20,11 +22,15 @@ app.use(bodyParser.json())
 // call other routes and filter paths
 app.use('/player', playerRoutes);
 
-// Connect to db
-// mongoose.connect('mongodb://localhost:27017', { useNewUrlParser: true, useUnifiedTopology: true })
+// app connects to the database when app starts up before it starts listning to incoming requests
+db.initDb((err, db) => {
```

Comment:
We're passing a callback function here. 
And this function has two parameters `err` and `db`. While we're using `err` here but we don't use `db`. We better to remove this second parameter here and on `initDb` as well if they're not used.